static/js/click_handlers.js

```diff
@@ -399,6 +399,11 @@ $(function () {
 
     });
 
+    // on the end of the modified-message fade in, remove the fade-in-message class.
+    $("body").one("webkitAnimationEnd oanimationend msAnimationEnd animationend", ".fade-in-message", function () {
```

Comment:
Also, probably this code belongs in `message_list_view.js`, not `click_handlers.js`, for proximity, right?